test/hotspot/jtreg/runtime/exceptionMsgs/ArrayIndexOutOfBoundsException/libArrayIndexOutOfBoundsExceptionTest.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Oracle copyright uses a two year format so this should be "2019, 2021,"

Reply:
Thanks @dholmes-ora for the review!
I updated copyright year in new commit.